README.md

```diff
@@ -129,8 +130,31 @@ config.middleware.use(IPinfoMiddleware, {
   http_client: "",
   countries: "",
   filter: "",
+  ipselector: XForwardedIPSelector,
 })
 ```
+### IP Selection Mechanism
+
+By default, the IP is used by ignoring the reverse proxies depending on whether we are behind a reverse proxy or not.
+
+Since the desired IP by your system may be in other locations, the IP selection mechanism is configurable and some alternative built-in options are available.
+Using built-in IP selectors
+
+  - [DefaultIPSelector](./lib/ipinfo-rails/ipselector/default_ipselector.rb)
+  - [XForwardedIPSelector](./lib/ipinfo-rails/ipselector/xforwarded_ipselector.rb)
+
+#### Using a custom IP selector
+
+In case a custom IP selector is required, you may implement the `IPSelectorInterface` and pass the class to `ipselector` in config.
```

Comment:
```suggestion
In case a custom IP selector is required, you may implement `IPSelectorInterface` and pass the class to `ip_selector` in config.

```